flink/src/test/java/org/apache/iceberg/flink/FlinkTestBase.java

```diff
@@ -53,4 +64,36 @@ public static void stopMetastore() {
     flinkCatalogs.values().forEach(Catalog::close);
     flinkCatalogs.clear();
   }
+
+  protected TableEnvironment getTableEnv() {
```

Comment:
Moving those tow method inside this super class, because the newly introduced unit test `TestFlinkHiveCatalog`  would use them.